docs/whats-new.md

```diff
@@ -1,9 +1,16 @@
 # react-map-gl v3.2
 
-Under development
+Realease date: January, 2018
+
+## Highlights
 
-- **Support for Map Reuse**: A new property `reuseMaps` is provided for applications that create and destroy maps, to help work around a mapbox-gl resource leak issue that can lead to a browser crash in certain situations.
```

Comment:
Thanks: We could mention it as experimental at the bottom of this list, and mark it as experimental in the docs above.